docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>This is overall really nice, I think it would be good to eventually have some more explanation with references to other places in the docs. After fixing the note issue, I think this is a good addition to the documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Okay, I have added a little introduction to the notebook. This notebook is not the final one, this was just a prototype for showing the visualization plot.